.github/workflows/test-all-packages.yml

```diff
@@ -510,7 +510,7 @@ jobs:
     # END-RESTORE-BOILERPLATE
 
     - name: yarn test (SwingSet)
```

Comment:
change the name to "... part 1: A-I"?

or add a comment explaining that swingset tests are broken into 3 parts alphabetically?
It was a bit tricky to get just from the code.

Reply:
Ah... I see you did add a comment in a later commit.